float64[21] pos
float64[21] loadServo
float64[21] outMimo
float64 selisih

